controller/tests/models/domain/rsptag_test.py

```diff
@@ -359,3 +390,16 @@ def test_from_str() -> None:
     # them in dictionary form.
     for tag, expected in test_cases.items():
         assert asdict(RSPImageTag.from_str(tag)) == expected
+
+
+def test_age() -> None:
+    rsptag = RSPImageTag.from_str("d_2021_05_27")
+    assert rsptag.date == datetime.datetime(
+        year=2021, month=5, day=27, tzinfo=datetime.UTC
+    )
+    orig_age = rsptag.age()
+    assert orig_age is not None
+    time.sleep(0.1)
+    new_age = rsptag.age()
+    assert new_age is not None
+    assert new_age > orig_age
```

Comment:
I don't think there's any reason to test that the date is changing, and that requires adding delays to the test suite. The important part is that the time depends on the basis passed in and that the math is correct. So, I think the right test here is to pass in a fixed datetime, assert the resulting age is the correct and expected timedelta, grab `datetime.now(tz=UTC)`, call `age()` with no arguments, and assert that the result is within a second of the timedelta you get when subtracting the earlier result of `datetime.now` from `rsptag.date`.